### PR TITLE
Add Silicon Valley 🙏

### PR DESCRIPTION
**Pull request by AdamEXu**
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏
Add Silicon Valley 🙏

(pretty pretty please?)

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/silicon-valley**](https://counterspell-a44w46ra9.hackclub.dev/silicon-valley) |
|:-:|
|<a href="https://counterspell-a44w46ra9.hackclub.dev/silicon-valley">Screenshot of /silicon-valley</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/TgP46xrla6NiuU9lDwmy3bpStbxDiOxnxwxzXFZclJC6GHYqfRruQUsIR0FHu6VjYetLS0QJ7CSiOTDsfWhQ7A.png">(view full size)</a> |







Commit 7881ce75c26645027dcff1377f5a349a1fb8d6d9 (https://counterspell-a44w46ra9.hackclub.dev).

File: public/siliconvalley/hero/bg1.png
Comment (samdev-7) [inline public/siliconvalley/hero/bg1.png:1]: These images are in the wrong place folder, please remove them.

File: public/city/Silicon-Valley/hero/bg1.png
Comment (samdev-7) [inline public/city/Silicon-Valley/hero/bg1.png:1]: For consistency sakes, could you rename the silicon-valley folder to be lowercase?

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost good to go!

File: public/city/Silicon-Valley/hero/bg1.png
Comment (AdamEXu) [reply public/city/Silicon-Valley/hero/bg1.png:1]: of course

**Comment by AdamEXu** [comment]
should be good now?

**Comment by AdamEXu** [comment]
@samdev-7

**Comment by AdamEXu** [comment]
🙏 pls 💖

**Comment by AdamEXu** [comment]
pwetty pwetty pwease ❤️

**Review by samdev-7 — APPROVED** [review]

**Comment by AdamEXu** [comment]
tysm

**Pull request merged**